### PR TITLE
Code samples update

### LINKED ISSUE #252: Update .code-samples.meilisearch.yaml
⚠️ This issue is generated. It means the examples and the namings do not necessarily correspond to the language of this repository.
Also, if you are a maintainer, feel free to add any clarification and instruction about this issue.

Sorry if this is already wholly/partially implemented. Feel free to let me know about the state of this issue in the repo.

Related to https://github.com/meilisearch/integration-guides/issues/185

----

Check the [integration-guides issue](https://github.com/meilisearch/integration-guides/issues/185) for more information and the complete description about what should be done here (each TODO item has its report in the #185).

TODO:

- [x] New code samples regarding the new getting started
- [x] Replace `description` into `overview` and `genre` into `genres`
- [x] Update geosearch guide
- [x] Remove unused code samples
- [x] Update the primary key examples
- [x] Add `settings_guide_sortable_1` code samples
- [x] Add `landing_getting_started_1` code samples

### PR DESCRIPTION
**Pull request by alallema**
closed #252

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -150,6 +153,69 @@ getting_started_search_md: |-
...
   ```
   [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
+getting_started_add_meteorites: |-
+  import json
+
+  json_file = open('meteorites.json')
+  meteorites = json.load(json_file)
+  client.index('meteorites').add_documents(meteorites)
+getting_started_update_rankingRules: |-
+  await client.Index("movies").UpdateRankingRulesAsync(new string[] {
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml]: is see the function have camelCase and snake-case is this normal?

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -150,6 +153,69 @@ getting_started_search_md: |-
...
+getting_started_configure_settings: |-
+  var newSettings = new Settings
+  {
+      FilterableAttributes = new string[] { "mass", "_geo" },
+      SortableAttributes = new string[] { "mass", "_geo" },
+  };
+  await client.Index("meteorites").UpdateSettingsAsync(newSettings);
+getting_started_communicating_with_a_protected_instance: |-
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "apiKey");
+  await client.Index("Movies").SearchAsync<Movie>("");
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml]:
```suggestion
  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "apiKey");
  
  await client.Index("Movies").SearchAsync<Movie>("");
```

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -169,10 +235,20 @@ faceted_search_facets_distribution_1: |-
...
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+
+  var documents = new Movie[] {
+      new Movie { Id = "1", Title = "Carol" },
+      new Movie { Id = "2", Title = "Wonder Woman" },
+      new Movie { Id = "3", Title = "Life of Pi" },
+      new Movie { Id = "4", Title = "Mad Max: Fury Road" },
+      new Movie { Id = "5", Title = "Moana", ", "Action" },
+      new Movie { Id = "6", Title = "Philadelphia" }
+  };
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:257]:
```suggestion
  };

```

File: .code-samples.meilisearch.yaml (hunk 1 above)
Comment (alallema) [reply .code-samples.meilisearch.yaml]: Absolutely not! Good catch

**Review by bidoubiwa — DISMISSED** [review]
🔥🔥🔥🔥 LGTM 🔥🔥🔥🔥

**Comment by alallema** [comment]
bors merge

**Review by brunoocasali — APPROVED** [review]
🎉 🦖

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**